products/jbrowse-cli/src/commands/text-index.ts

```diff
@@ -90,7 +90,7 @@ export default class TextIndex extends JBrowseCommand {
         for (const x in adapters) {
           if (adapters[x].textSearchAdapterId === hash) {
             adapters[x].ixFilePath.uri = `trix/${asm}.ix`
-            adapters[x].ixFilePath.uri = `trix/${asm}.ixx`
+            adapters[x].ixxFilePath.uri = `trix/${asm}.ixx`
```

Comment:
this is a notable change that is maybe a typo on the current text_searching branch

Reply:
the change was added to text_searching directly:)